### ISSUE TITLE
The settings-menu is not shown when the app is full screen

### ISSUE THREAD

**Issue by joseseie** [quasarframework/quasar-ui-qmediaplayer#335]
**Describe the bug**

The settings-btn is actually showing, but when I click, I see it was clicked and when I inspect the DOM, I can see the setting-menu being overlaid by other layout elements.

**To Reproduce**
Click on the fullscreen option, and click on settings-btn, in fullscreen mode the menu with the settings options will not be shown.

**Expected behavior**
Clicking on the settings button was supposed to see the menu with the settings options.

**Screenshots**
![Captura de ecrã 2021-10-28, às 11 03 15](https://user-images.githubusercontent.com/18400142/139225412-0a552f4d-b48f-40cf-a47a-1dfecf5f68d3.png)

![Captura de ecrã 2021-10-28, às 11 02 14](https://user-images.githubusercontent.com/18400142/139225389-0df7c1c4-86f4-45d7-8702-11935291f9e4.png)

**Desktop (please complete the following information):**
 - OS: [MacOs Big Sur, v11.4]
 - Browser [chrome]

**Additional context**
```
SeieMacOs:App joseseie$ quasar info

Operating System - Darwin(20.5.0) - darwin/x64
NodeJs - 14.17.1

Global packages
  NPM - 7.22.0
  yarn - 1.22.10
  @quasar/cli - 1.0.0-rc.2
  cordova - 10.0.0

Important local packages
  quasar - 1.10.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 1.8.5 -- Quasar Framework local CLI
  @quasar/extras - 1.11.1 -- Quasar Framework fonts, icons and animations
  vue - 2.6.11 -- Reactive, component-oriented view layer for modern web interfaces.
  vue-router - 3.1.6 -- Official router for Vue.js 2
  vuex - 3.3.0 -- state management for Vue.js
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed
  @babel/core - 7.9.6 -- Babel compiler core.
  webpack - 4.43.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 3.10.3 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - 4.3.1 -- A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.
  register-service-worker - 1.7.1 -- Script for registering service worker, with hooks

Quasar App Extensions
  @quasar/quasar-app-extension-qmediaplayer - 1.4.1 -- A Quasar App Extension for @quasar/quasar-ui-qmediaplayer

Networking
  Host - SeieMacOs
  en0 - 192.168.43.252

SeieMacOs:App joseseie$

```

**Comment by hawkeye64** [comment]
Hi, thanks for reporting this. I'll look into it as soon as I can.

**Comment by hawkeye64** [comment]
Fix is available in v2.0.0-beta.3

**Comment by joseseie** [comment]
> Fix is available in v2.0.0-beta.3

Thanks @hawkeye64
